TinCan/ActivityDefinition.cs

```diff
@@ -26,13 +30,13 @@ public class ActivityDefinition : JsonModel
         public LanguageMap name { get; set; }
         public LanguageMap description { get; set; }
         public Extensions extensions { get; set; }
-        //public InteractionType interactionType { get; set; }
-        //public List<String> correctResponsesPattern { get; set; }
-        //public List<InteractionComponent> choices { get; set; }
-        //public List<InteractionComponent> scale { get; set; }
-        //public List<InteractionComponent> source { get; set; }
-        //public List<InteractionComponent> target { get; set; }
-        //public List<InteractionComponent> steps { get; set; }
+        public InteractionType interactionType { get; set; }
+        public List<String> correctResponsesPattern { get; set; }
+        public List<InteractionComponent> choices { get; set; }
+        public List<InteractionComponent> scale { get; set; }
+        public List<InteractionComponent> source { get; set; }
+        public List<InteractionComponent> target { get; set; }
+        public List<InteractionComponent> steps { get; set; }
```

Comment:
It will probably become a need pretty quickly to allow for individual additions of a single `InteractionComponent` to each of these lists. The public interface of the class requires you to set the whole list. That's been an issue elsewhere in the library in the past. Not something that has to hold up this PR, but it is something to be aware of for the future.